third_party/move/move-compiler-v2/src/experiments.rs

```diff
@@ -4,7 +4,7 @@
 
 /// Container for experiments in the compiler. Those can be activated
 /// via the `--experiment=<name>` flag. One can also use the env var
-/// `MOVE_COMPILER_EXP=<comma list>` to activate those flags.
+/// `MVC_EXP=<comma list>` to activate those flags.
```

Comment:
If you change compiler controls, please update the best doc we have currently: https://www.notion.so/aptoslabs/Move-Compiler-Debugging-Tips-322ae742b44846da840eec0f4e195a0b#023e370ac7264eebb4215d42d3e545b7

which should be cleaned up and put into a README, I guess.  (I filed an issue to track that https://github.com/aptos-labs/aptos-core/issues/12163).

Reply:
Really should be in markdown than this would be smoother. In any case, the document is in `experiment.rs` and has been updated, the other doc should just point to it I think.

Reply:
Your irony doesn't make a lot of sense here. The source of truth is documentation on code or markdown files. Experiments are a standard concept of our compiler and they are documented in this file. There is not need to go of thousands of lines of code since this here has only a few dozens here.